### PR TITLE
Add September 2018 scala meetup Eff talk

### PR DESCRIPTION
**Pull request by jcarrag**
Adding my slides for the September Tokyo Scala Meetup

### CODE REVIEW

**Comment by gvolpe** [comment]
@jcarrag  I have created a generic template to unify all the slides. Take a look at how the current slides were modified here -> https://github.com/paidy/talks

You basically need to remove all the assets (css, js, plugin,etc), copy / paste the files `Grunt.js` and `package.json` from the template and update the links in your `index.html`.

File: 2018-09-18/tokyo-meetup-eff/Gruntfile.js [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+			files: [ 'test/*.html' ]
+		},
+
+		uglify: {
+			options: {
+				banner: '<%= meta.banner %>\n',
+				screwIE8: false
+			},
+			build: {
+				src: 'js/reveal.js',
```
Comment (gvolpe) [inline 2018-09-18/tokyo-meetup-eff/Gruntfile.js]: All the resources should be pointing to `https://rawgit.com/paidy/talks/master/static/`. I believe you have the old `Grunt.js` file.

File: 2018-09-18/tokyo-meetup-eff/bower.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "reveal.js",
```
Comment (gvolpe) [inline 2018-09-18/tokyo-meetup-eff/bower.json]: I don't think this file is needed?

File: 2018-09-18/tokyo-meetup-eff/package.json [hunk 3]
```diff
@@ -0,0 +1,43 @@
+{
+  "name": "reveal.js",
+  "version": "3.6.0",
+  "description": "The HTML Presentation Framework",
+  "homepage": "http://revealjs.com",
+  "subdomain": "revealjs",
+  "main": "js/reveal.js",
```
Comment (gvolpe) [inline 2018-09-18/tokyo-meetup-eff/package.json]: https://rawgit.com/paidy/talks/master/static/

File: 2018-09-18/tokyo-meetup-eff/Gruntfile.js (hunk 1 above)
Comment (jcarrag) [reply 2018-09-18/tokyo-meetup-eff/Gruntfile.js]: Thanks, I haven't yet merged the new project structure changes into this PR. I'm doing it now.

**Pull request merged**